src/Artsy/Analytics/trackingMiddleware.ts

```diff
@@ -43,13 +44,34 @@ export function trackingMiddleware(options: TrackingMiddlewareOptions = {}) {
           if (!foundExcludedPath) {
             logger.warn("Tracking PageView:", pathname)
 
-            analytics.page(
-              { path: pathname, referrer },
-              { integrations: { Marketo: false } }
-            )
+            const trackingData: {
+              path: string
+              referrer?: string
+              url: string
```

Comment:
It doesn't look like `url` was provided at all before. Does that automatically come thru as a relative path, and that's no good anymore? (and we prefer a full URL)

Reply:
`url` is automatically inferrered by segment if not provided. The issue is that we were firing the tracking call before the url was completely transitionioned, so segment would immediately invoke the window.location object and populate.

Reply:
Ah ok, saw your other comment too. The fact the Segment lib automatically infers and includes this (if not specified) makes sense with this change, I hadn't realized that.